src/main/java/uk/gov/hmcts/reform/civil/callback/OrderCallbackHandler.java

```diff
@@ -28,11 +28,11 @@ protected OrderCallbackHandler(DashboardApiClient dashboardApiClient, DashboardN
     }
 
     protected boolean isEligibleForReconsideration(CaseData caseData) {
-        return caseData.isSmallClaim()
+        return featureToggleService.isCaseProgressionEnabledAndLocationWhiteListed(caseData.getCaseManagementLocation().getBaseLocation())
```

Comment:
I'm not sure that's enough - don't we also need to change the claimant/defendant handler checks for this?:
**isSDODrawnPreCPRelease()** which exists in the child classes for claimant/defendant.

Reply:
Done